docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
 
-`DeepReg` provides command-line tool options that implement basic functionalities including model training, prediction and image warping. The yaml files and command-line arguments are used as the basis of configuring the various options with `DeepReg`. This design choice has been motivated by both research and education purposes originally set out in the `DeepReg` project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research topics. A MICCAI Educational Challenge has utilised the `DeepReg` code and demos to explore the link between classical algorithms and deep-learning-based methods, while a latest research paper investigated temporal changes for prostate cancer patients under active surveillance programme by using longitudinal image registration, adapted from the `DeepReg` code [@yang2020longitudinal].
+DeepReg provides command-line tool options that enable a set of basic and advanced functionalities for model training, prediction and image warping. The yaml files and command-line arguments are used for configuring the various options with DeepReg. This design choice has been motivated by both research and education purposes originally set out in the DeepReg project, which aims an open and easy access to the benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research advances. A MICCAI Educational Challenge has utilised the DeepReg code and demos to explore the link between classical algorithms and deep-learning-based methods, while a latest research paper investigated temporal changes for prostate cancer patients under active surveillance programme, by using longitudinal image registration adapted from the DeepReg code [@yang2020longitudinal].
```

Comment:
"which aims an open and easy access" -> "which aims for open and easy access", the access is not countable

Reply:
"These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research advances." personally i found the second part "rapid utilising ..." a bit hard to understand, tho i do not have any suggestion :(

Reply:
"while a latest research paper" -> "while a recently published paper", i mean here paper must be research paper? and grammarly is saying it has to be "the latest" or you can say "one of the latest papers"